bindings/python/htcondor2/_completion_type.py

```diff
@@ -1 +1,28 @@
+import sys
 
+if sys.version_info < (3, 5, 0):
+    import compat_enum as enum
+else:
+    import enum
+
+
+class CompletionType(enum.IntEnum):
+    """
+    .. attribute:: Nothing
+    .. attribute:: Resume
+
+    Added in 8.9.11:
+
+    .. attribute:: Exit
+    .. attribute:: Restart
+
+    Added in 9.11.0:
+
+    .. attribute:: Reconfig
+    """
```

Comment:
What is the CompletionType enum represent. Perhaps add a blurb to the Doc string

Reply:
In progress.  (I don't know what `reconfig` actually does.)

Reply:
... and done.